KOGITO-6308 - SAGA SW example (#4096)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/ReleaseNotesKogito.1.16.0.Final/ReleaseNotesKogito.1.16.0.Final.adoc

== {PRODUCT} runtimes

> === {PRODUCT} Serverless Workflow example demonstrating SAGA pattern

Reviewer: ```suggestion
=== Example application for {PRODUCT} Serverless Workflow to demonstrate SAGA pattern
```

> In {PRODUCT} 1.16.0, a new example has been added to demonstrated how to use Serverless Workflow can be used to implement SAGA pattern.

Reviewer: ```suggestion
In {PRODUCT} 1.16.0, a new example application is added to demonstrate how to use Serverless Workflow to implement the SAGA pattern.
```

> The example is based on Quarkus runtime and is available at https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-saga-quarkus[serverless-workflow-saga-quarkus].

Reviewer: ```suggestion
The https://github.com/kiegroup/kogito-examples/tree/stable/kogito-quarkus-examples/serverless-workflow-saga-quarkus[serverless-workflow-saga-quarkus] example application is based on Quarkus runtime.
```
